docs/chemconverter/index.mdx

```diff
@@ -105,8 +105,27 @@ Currently there are four used classes:
 | PEAK TABLE |                                                                                                                                              |                                                                                  |
 | NTUBLES    |                                                                                                                                              |                                                                                  |
 
+##### Data Type
+
+A drop-down list of ontologies (mostly chemical analytic techniques) defining the kind of layout for displaying and plotting curves and spectra in ChemSpectra.
+Not all ontologies known in the ELN are findable here because it's only for layout purposes.
+If you think that something is missing, feel free to tell us.
+
+##### X & Y Units
+
+A list of physical units containing the most common units with and without prefixes.
+If you need additional units, please tell us or use the "scalar operation" feature to convert the column values.
+
+##### Table columns
+
+A drop-down menu for defining the x and y column values while a preview can be seen on the end of the left side as "Input table data".
+Currently only 2D data extraction is possible so if you need more, you can create additional Output tables and merge them together with external tools.
+Alternative you can use the "column operation" feature to add, subtract, divide or multiply columns together.
+
+In a future version it will become possible to create profiles without Output tables if you intend to use the converter only for Metadata extraction like described below.
```

Comment:
What section does "like described below" refer to? Should there be a link?

Reply:
Later there will be a section named sth. like "Metadata extraction" down below.
But it's not written yet.

Reply:
I removed the text that suggested documentation is available. Please add it again once the documentation is included.